File: FoosballGames/FoosballGamesContext.cs
using Microsoft.EntityFrameworkCore;

namespace FoosballGames
{
    public class FoosballGamesContext : DbContext
    {
        public FoosballGamesContext(DbContextOptions options) : base(options)
        {
        }

        public DbSet<DbFoosballGame> FoosballGames { get; set; } = null!;

        protected override void OnModelCreating(ModelBuilder modelBuilder)
        {
            base.OnModelCreating(modelBuilder);

            modelBuilder.HasDefaultSchema("FoosballGames");

            modelBuilder.Entity<DbFoosballGame>(m =>
            {
                m.HasKey(s => s.Id);
                m.Property(a => a.JsonContent).IsRequired().HasColumnType("json");
            });
        }
    }
}

File: FoosballGames.Db/FoosballGamesContext.fs
namespace FoosballGames

open FoosballGames
open Microsoft.EntityFrameworkCore

type FoosballGamesContext(options: DbContextOptions) =
    inherit DbContext(options)
    
    [<DefaultValue>] val mutable foosballGames: DbSet<DbFoosballGame> 
    member this.FoosballGames
        with get () = this.foosballGames
        and set value = this.foosballGames <- value
    
    override this.OnModelCreating(builder: ModelBuilder) =
        base.OnModelCreating(builder)
        builder
            .HasDefaultSchema("FoosballGames")
            .Entity<DbFoosballGame>(
                fun m ->
                    m.HasKey(fun x -> x.Id :> obj) |> ignore
                    m.Property(fun s -> s.JsonContent :> obj).HasColumnType("json") |> ignore
                )
            |> ignore